src/arkode/arkode_mristep.c

```diff
@@ -2864,6 +4448,62 @@ int mriStepInnerStepper_Reset(MRIStepInnerStepper stepper, sunrealtype tR,
   }
 }
 
+/* Gets the inner (fast) stepper accumulated error */
+int mriStepInnerStepper_GetError(MRIStepInnerStepper stepper,
+                                 sunrealtype* accum_error)
+{
+  if (stepper == NULL) { return ARK_ILL_INPUT; }
+  if (stepper->ops == NULL) { return ARK_ILL_INPUT; }
+
+  if (stepper->ops->geterror)
+  {
+    stepper->last_flag = stepper->ops->geterror(stepper, accum_error);
+    return stepper->last_flag;
+  }
+  else
+  {
+    /* assume stepper provides exact solution */
+    *accum_error = SUN_RCONST(0.0);
+    return ARK_SUCCESS;
```

Comment:
I'm good with emulating how `SUNStepper` handles this; if a user implements an exact solution then they should proudly declare an accumulated error of zero, instead of leaving the function unimplemented.